components/dropdown/dropdown-content-styles.js

```diff
@@ -2,9 +2,14 @@ import '../colors/colors.js';
 import { css } from 'lit-element/lit-element.js';
 
 export const dropdownContentStyles = css`
+
 	:host {
 		--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation;
 		--d2l-dropdown-animation-name: d2l-dropdown-animation;
+		--d2l-dropdown-mobile-tray-right-name: d2l-dropdown-mobile-tray-right;
```

Comment:
I think I must be missing something - wondering why these are variables instead of just specifying these values where they are used...?

Reply:
They are so you can override the animation depending on other css variables, ie. override the default dropdown animation for dark themes. You are right, I can remove these.

Reply:
Cool yeah... I'd only expose stuff as variables that can be overridden if needed.